ui/ui-app/src/main/resources/static/js/ops-mgr/service-health/service-detail.html

```diff
@@ -110,7 +110,7 @@
                         <td md-cell>
                             {{component.alerts.length}}
                         </td>
-                        <td md-cell hide show-gt-md >{{component.checkDate | date : "MM/dd/yyyy 'at' h:mm"}}</td>
+                        <td md-cell hide show-gt-md >{{'service-component-detail.time-locale' | translate:{date:component.checkDate} }} {{"service-component-detail.at" | translate}} {{'service-component-detail.time-locale2' | translate:{date:component.checkDate} }}</td>
```

Comment:
There are no English translations for `service-component-detail.at` and `service-component-detail.time-locale2`. Please fix this to only use `service-component-detail.time-locale`.